crates/cheatnet/src/rpc.rs

```diff
@@ -83,6 +83,7 @@ type SyscallSelector = DeprecatedSyscallSelector;
 pub enum CallContractOutput {
     Success { ret_data: Vec<Felt252> },
     Panic { panic_data: Vec<Felt252> },
+    Error { msg: String },
```

Comment:
This will cause a conflict with #305 assert macros.

Reply:
Which ones specifically? I think all tests are passing now

Reply:
It's already fixed